pkg/provider/k8s/k8s.go

```diff
@@ -416,7 +433,49 @@ func (c *K8s) deploymentApply(resource runtime.Object) error {
 		func() (bool, error) { return c.deploymentReady(resource) })
 }
 
+func (c *K8s) customResourceDefinitionApply(resource runtime.Object) error {
```

Comment:
```suggestion
func (c *K8s) customResourceApply(resource runtime.Object) error {
```

Lets keep the names as short as possible. the same for Delete - `customResourceDelete`